test/pool-utils/PoolTestFixture.ts

```diff
@@ -333,6 +368,48 @@ export abstract class PoolTestFixture {
     return state;
   }
 
+  /**
+   * TESTING UTILITY checks user state for TPS+TYS balance and YBT balance
+   * @param user User whose wallet to check
+   * @param shares Amount of TPS and TYS we expect
+   * @param ybt Amount of YBT we expect
+   * @param message Description of what we expected to happen
+   */
+  async checkWallet(user:Signer, shares:ShareExpectation, ybt:YBTExpectation, message?:string): Promise<void> {
+    const us = await this.userState(user);
+    us.expectMulti(shares.tps, shares.tys, ybt.pegged, ybt.pegged, message);
```

Comment:
Hmm, is this correct or wrong?

```suggestion
    us.expectMulti(shares.tps, shares.tys, ybt.pegged, ybt.unpegged, message);
```

Reply:
This is what I get when I review commit-by-commit.

Reply:
Yeah, I made a mistake in the original commit, was fixed in the next commit